rllib/train.py

```diff
@@ -144,10 +144,10 @@ def run(args, parser):
                 num_gpus=args.ray_num_gpus or 0,
                 object_store_memory=args.ray_object_store_memory,
                 redis_max_memory=args.ray_redis_max_memory)
-        ray.init(redis_address=cluster.redis_address)
+        ray.init(address=cluster.redis_address)
     else:
         ray.init(
-            redis_address=args.redis_address,
+            address=args.ray_address,
```

Comment:
can you also change all of the examples in `tune` and `rllib`?

Reply:
I'm purposely not changing them since I don't want users to start using it on old versions of Ray. We can add it to the docs in a couple months.